qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1393,7 +1400,6 @@ def __str__(self):
             ret += f"\nBackend: {self.backend}"
         if self.tags:
             ret += f"\nTags: {self.tags}"
-        ret += f"\nData: {len(self._data)}"
```

Comment:
I think it's useful to know how much data is in the container (though I removed this whole repr method from the base DB class in another PR anyway, since if we want a user to only use ExperimentData we only need a pretty print for that class)

Reply:
I returned it, but I think the phrasing should be clearer